synthesis/tests/klayout.lyt

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is there a way we can get this from the pdk deps?

Reply:
In skywater-pdk repositories there are no KLayout files. Commit messages in https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/299 indicate that KLayout files were taken from [RTimothyEdwards/open_pdks](https://github.com/RTimothyEdwards/open_pdks) and those in open_pdks are taken from [mabrains/sky130_klayout_pdk](https://github.com/mabrains/sky130_klayout_pdk).

We could possibly add  `mabrains/sky130_klayout_pdk` here as dependency and write some new rules for fetching this file. Please let me know what do you think about that.

Reply:
Klayout *.lyt file has 3 different sources from which it could be imported to [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/platforms/sky130hd/sky130hd.lyt):
* [mabrains/sky130_klayout_pdk](https://github.com/mabrains/sky130_klayout_pdk/blob/main/sky130_tech/tech/sky130/sky130.lyt)
* [efabless/sky130_klayout_pdk](https://github.com/efabless/sky130_klayout_pdk/blob/main/sky130_tech/tech/sky130/sky130.lyt)
* [RTimothyEdwards/open_pdks](https://github.com/RTimothyEdwards/open_pdks/blob/master/sky130/klayout/sky130.lyt)

I tested all of them and compared against *.lyt from `ORFS` and it looks like the most similar is the file from `open_pdks`. There are also some differences in GDS files generated with those KLayout configurations. The ones generated from configurations taken from `ORFS` and `open_pdks` look alright but those generated with configurations from `efabless` and `mabrains` do not have any paths between cells on layers `met2`, `met3`, `met4`

So `open_pdks` looks like a good candidate for fetching those files from dependencies. But there is a catch: open_pdks does not support ASAP7 and doesn't store any files for this technology - ORFS does. If we want to get `fill_config.json` and `klayout.lyt` from dependencies then I think ORFS would be the best repository for fetching this kind of data.